pkg/gofr/gofr.go

```diff
@@ -88,6 +88,17 @@ func New() *App {
 	app.httpServer.certFile = app.Config.GetOrDefault("CERT_FILE", "")
 	app.httpServer.keyFile = app.Config.GetOrDefault("KEY_FILE", "")
 
+	// Add Default routes
+	app.add(http.MethodGet, "/.well-known/health", healthHandler)
+	app.add(http.MethodGet, "/.well-known/alive", liveHandler)
+	app.add(http.MethodGet, "/favicon.ico", faviconHandler)
+
+	if _, err = os.Stat("./static/openapi.json"); err == nil {
+		app.add(http.MethodGet, "/.well-known/openapi.json", OpenAPIHandler)
```

Comment:
Why not using the default swagger constant you defined? For these 2 lines.

Reply:
Sure used the constant here